app/Services/Interventions/MaintenanceContractService.php

```
@@ -143,33 +155,33 @@ protected function sendReminderIfDue(MaintenanceContract $contract, CarbonInterf
-
         $recipient = $this->resolveRecipient($contract);
 
         if (blank($recipient)) {
             return false;
         }
 
-        Notification::route('mail', $recipient)
-            ->notify(new MaintenanceContractReminderNotification($contract));
-
-        MaintenanceContractReminder::firstOrCreate([
+        $reminder = MaintenanceContractReminder::firstOrCreate([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Interventions\MaintenanceContractReminder::firstOrCreate().


(staticMethod.notFound)